DashKitUI/View Models/DashSettingsViewModel.swift

```diff
@@ -49,3 +117,26 @@ extension DashPumpManager {
         }
     }
 }
+
+extension DashPumpManager: PodDetails {
+    var podIdentifier: String {
+        return podId ?? "None"
+    }
+    
+    var lot: String {
+        return "TODO"
```

Comment:
[nit] Maybe add `// TODO` comments as well?  Xcode makes it easy to find those...

Reply:
This is not a reminder for me; more a placeholder. Could change to TBD; will be tackled in https://tidepool.atlassian.net/browse/LOOP-1174